lang/system/pl.xml

```diff
@@ -3,35 +3,83 @@
 <lang>
 	<main_menubar>
 		<file name="Plik">
-			<open_pref_path>Open Pref Path</open_pref_path>
-			<install_firmware>Install Firmware</install_firmware>
+			<open_pref_path>Otwórz preferowaną ścieżkę emulatora</open_pref_path>
+			<open_textures_path>Otwórz ścieżkę folderu z teksturami</open_textures_path>
+			<install_firmware>Zainstaluj oprogramowanie</install_firmware>
 			<install_pkg>Zainstaluj .pkg</install_pkg>
-			<install_zip>Instalacja .zip, .vpk</install_zip>
+			<install_zip>Zainstaluj .zip, .vpk</install_zip>
 			<install_license>Zainstaluj licencję</install_license>
 			<exit>Zamknij</exit>
 		</file>
-		<emulation name="Emulation">
-			<last_apps_used>Last Apps used</last_apps_used>
+		<emulation name="Emulacja">
+			<last_apps_used>Ostatnio używane aplikacje</last_apps_used>
 		</emulation>
-		<configuration name="Configuration">
+		<debug name="Debugowanie">
```

Comment:
here is not used the "name"

Reply:
@Croden1999 it's used indeed, shows up in menubar translated.

Reply:
but i already cancel this title in pre version

Reply:
Okay whatever, this project will be polinglish anyways, could blindly copy english xml and not dealing with this out random, waiting for a merge.

Removed in the name of Winnie the Pooh.
![image](https://github.com/Vita3K/Vita3K/assets/47083527/82e3c605-2435-4877-b535-b29c6bce5a47)
W IMIĘ POLSKI LUDOWEJ!
Signing off, comrade! o7